.goreleaser.yaml

```diff
@@ -17,7 +17,7 @@ archives:
 checksum:
   name_template: 'checksums.txt'
 snapshot:
-  name_template: "{{ incpatch .PreviousTag }}-dev+{{ .ShortCommit }}"
+  name_template: "{{ incpatch .PreviousTag }}+dev.{{ .ShortCommit }}"
```

Comment:
Pretty sure you want `Version` and not `PreviousTag`. Leading `v` is a bit of a no-no...